backend/databasemodel/model.sql

```diff
@@ -1281,13 +1284,18 @@ SET timezone = 'Europe/Helsinki';
 INSERT INTO lipas.metadata (
 	type_codes_summer,
 	type_codes_winter,
-	type_codes_all_year
+	type_codes_all_year,
+	tarmo_categories_summer,
```

Comment:
Where is this data used? We already use type_codes_summer, type_codes_winter and type_codes_all_year list to set the season in each lipas object.

This seems to be a parallel mapping that tells us the season based on the tarmo category string.

Why do we need this mapping, since tarmo category string will be added based on the type code anyway?

Reply:
True. This is kind of duplicate thing since only tarmo category values whose visibility we want to change according to the current season are Lipas features. I will remove these fields.